framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2061,6 +2061,75 @@ protected void adaptGetClassReturnTypeToReceiver(
         classWildcardArg.setExtendsBound(newBound);
     }
 
+    /**
+     * Return the element type of the given iterator. This is usually the type of {@code
```

Comment:
@mernst I find the use of `iterator` here confusing.  If not an array, the expression must implement `Iterable` not `Iterator`.  I think you mean `iterator` as in an expression that can appear in a for each loop, but it's not clear.

Reply:
You are right:  that should have been iterable, not iterator.  Can you change that and make any other documentation improvements that you find needful?